packages/sdk-v2/src/sdk/base/index.ts

```diff
@@ -1,4 +1,4 @@
-export * from './hooks';
-export * from './react';
-export * from './sdk';
-export * from './utils';
+export * from "./hooks";
```

Comment:
why are there so many unrelated files quotes changes? what's the standard we use?

Reply:
we didn't have any. eslint was set to double, ctix was set to single

this is aligning the quotes to double everywhere (maybe could do a commit for it on master branch to clean this PR a bit up)